app/views/login.erb

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
this file is probably rendered within the layout which mean you have two DOCTYPEs and thats bad :P